govcd/vdcComputePolicy.go

```diff
@@ -0,0 +1,246 @@
+package govcd
+
+import (
+	"fmt"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"github.com/vmware/go-vcloud-director/v2/util"
+	"net/http"
+	"net/url"
+)
+
+type VdcComputePolicy struct {
```

Comment:
I believe it would be helpful to have somewhere in the code a comment that this compute policy in API corresponds to sizing policy in UI, so people don't need to guess that.

Reply:
added